spark/README.md

```diff
@@ -0,0 +1,5 @@
+Spark Habitat plan
+==================
+
+Run this package using `--topology=leader` in order to run some masters and
```

Comment:
Should be "master" and "slave" (singular), but using "leader/follower" terminology is preferred unless absolutely necessary.